server/src/routes/api/v1/activityReviewRouter.js

```diff
@@ -22,4 +22,14 @@ activityReviewRouter.post("/", async (req, res) => {
     }
 })
 
+activityReviewRouter.delete("/:id", async (req, res) => {
+    try {
+        const reviewToDelete = await Review.query().findById(req.params.id)
```

Comment:
we need to verify before delete that the user of the review is the same as req.user, that way we are validating for that information each step of the way (front end and back end)